azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/parser/AbstractConfigParser.java

```diff
@@ -102,6 +108,19 @@ protected void validate(Supplier<String> validator) throws AzureExecutionExcepti
         }
     }
 
```

Comment:
why you filter isExternalResource in convertResourcesToArtifact, it is not expected according to the method name.

Reply:
Renamed to `parseArtifactsFromResources`